apps/plea/email.py

```diff
@@ -43,12 +43,17 @@ def send_plea_email(context_data):
 
     context_data: dict populated by form fields
     """
+
     try:
-        court_obj = Court.objects.get_by_urn(context_data["case"]["urn"])
-    except Court.DoesNotExist:
-        logger.error("URN does not have a matching Court entry: {}".format(
-            context_data["case"]["urn"]))
-        raise
+        case = Case.objects.get(
+            urn__iexact=context_data["case"]["urn"].upper(), sent=False)
+
+    except (Case.DoesNotExist, Case.MultipleObjectsReturned):
```

Comment:
this is probably not right. if multiples are found, you just want to return the first one?